src/env/node/git/sub-providers/remotes.ts

```diff
@@ -38,8 +38,11 @@ export class RemotesGitSubProvider extends RemotesGitProviderBase implements Git
 		let remotesPromise = this.cache.remotes?.get(repoPath);
 		if (remotesPromise == null) {
 			async function load(this: RemotesGitSubProvider): Promise<GitRemote[]> {
+				const ci = await this.container.cloudIntegrations;
+				const connections = await ci?.getConnections();
```

Comment:
This is an API call to our backend. Have we checked to make sure this isn't happening very frequently, especially when scaled across all GL clients that update to this version?